src/js/components/ThumbsRating/ThumbsRating.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Like, LikeFill, Dislike, DislikeFill } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+export const ThumbsRating = ({
+  fillColor,
+  outlineColor,
+  formProps,
+  name,
+  label,
+  options,
+  ...rest
+}) => {
+  const likeOption = options[0];
```

Comment:
should we have some default options, in the case that the user doesn't provide any?

Reply:
yes I can add 
thought I did but might of been in other PR